app/frontend/src/aquifers/components/Search.vue

```diff
@@ -16,19 +16,21 @@
   <b-card no-body class="p-3 mb-4">
     <h5>Aquifer Search</h5>
```

Comment:
@stephenhillier we're going to have to assess these tags throughout the site at some point. (re. accessibility)